modules/xml/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(xml_HEADERS
+    "xml-parser.h"
+    "${CMAKE_CURRENT_BINARY_DIR}/xml-grammar.h"
+)
+
+set(xml_SOURCES
+    "xml-plugin.c"
+    "xml-parser.c"
+    "${CMAKE_CURRENT_BINARY_DIR}/xml-grammar.c"
+)
+
+generate_y_from_ym(modules/xml/xml-grammar)
+
+bison_target(xmlGrammar
+    ${CMAKE_CURRENT_BINARY_DIR}/xml-grammar.y
+    ${CMAKE_CURRENT_BINARY_DIR}/xml-grammar.c
+COMPILE_FLAGS ${BISON_FLAGS})
+
+option(ENABLE_xml "Enable xml ON")
```

Comment:
The description is confusing, because it says "Enable xml ON", but the default value for `option` is *OFF*. It is not stated here otherwise. I would follow the pattern:

```cmake
option(ENABLE_XML, "Enable xml module", ON)
```

Reply:
We decided as this is non-critical issue and general in other modules, this will not be scope of this patch.